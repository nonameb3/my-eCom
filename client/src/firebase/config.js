// firebase private config Firebase SDK snippet
export const config = {
  apiKey: "xxxxxxxxxxxxxxx",
  authDomain: "xxxxxxxxxxxxxxx",
  databaseURL: "xxxxxxxxxxxxxxx",
  projectId: "xxxxxxxxxxxxxxx",
  storageBucket: "xxxxxxxxxxxxxxx",
  messagingSenderId: "xxxxxxxxxxxxxxx",
  appId: "xxxxxxxxxxxxxxx",
  measurementId: "xxxxxxxxxxxxxxx"
};
