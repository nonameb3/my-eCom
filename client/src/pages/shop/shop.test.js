import React from "react";
import { BrowserRouter } from 'react-router-dom';
import { Provider } from "react-redux";
import config from "redux-mock-store";
import { shallow, mount } from "enzyme";

import Shop from "./shop.component";
import CollectionOverviewContainer from "../../components/collection-overview/collection-overview.component";
import CategoryContainer from "../category/category.component";

const mockStore = config();
const mockItems = [{ id: 1 }, { id: 2 }, { id: 3 }];
const store = mockStore({
  shop: {
    isFetching: false,
    collection: {
      test: { items: mockItems, title: "test" }
    }}
});

let wrapper;
describe("Shop compoent", () => {
  beforeEach(() => {
    wrapper = shallow(
      <Provider store={store}>
        <Shop />
      </Provider>
    );
  });

  it("should rander shop compoent", () => {
    expect(wrapper).toMatchSnapshot();
  });
});

/*
it("should rander shop compoent with CollectionOverviewContainer", () => {
  const mockProps = { match: {path: '/shop'}}
  wrapper = mount(
    <Provider store={store}>
      <BrowserRouter>
        <Shop {...mockProps}/>
      </BrowserRouter>
    </Provider>
  );

  expect(wrapper.find(CollectionOverviewContainer).exists()).toEqual(true);
});

it("should rander shop compoent with CategoryContainer", () => {
  const mockProps = { match: {path: '/shop/id'}}
  wrapper = mount(
    <BrowserRouter>
      <Provider store={store}>
        <Shop {...mockProps}/>
      </Provider>
    </BrowserRouter>
  );

  expect(wrapper.find(CategoryContainer).exists()).toEqual(true);
});
*/
