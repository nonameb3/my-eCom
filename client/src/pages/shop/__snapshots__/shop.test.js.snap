// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shop compoent should rander shop compoent 1`] = `ShallowWrapper {}`;
